ES/ESmy028.xml

```diff
@@ -89,7 +89,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
```

Comment:
the nesting is all wrong
please nest correctly

msItem ms_i1.1 (Introduction) must close here (?)
add `</msItem>` (and delete one closing tag at the end at line 622)

Reply:
ok, hope it is correct now.